samples/nrf9160/modem_shell/prj.conf

```diff
@@ -136,9 +135,11 @@ CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
 #CONFIG_MULTICELL_LOCATION_POLTE_CUSTOMER_ID="customer id"
 #CONFIG_MULTICELL_LOCATION_POLTE_API_TOKEN="api token"
 
+# nRF Cloud A-GPS, disable this in case you want to enable SUPL below
+CONFIG_NRF_CLOUD_AGPS=y
```

Comment:
This is there as part of MOSH "location" command PR so should be removed from here.
https://github.com/nrfconnect/sdk-nrf/pull/6086

Reply:
Yes, but I moved it a bit lower to get the A-GPS related configurations next to each other and added the comment. Should we have this with the other nRF Cloud options before multicell configuration or here, what do you think?

Reply:
Didn't see that :) Yeah, I think it would be nice to have nrf cloud items together.

Reply:
As discussed, moved the nRF Cloud A-GPS configuration with other nRF Cloud configurations and SUPL after those.